### PR TITLE
Start some internazionalization and add some feature to the  collection

### PR DESCRIPTION
**Pull request by p4535992**
- Start some internazionalization with a helper class
- Compress the code on the class 'effect-definitons' (more easy to read)
- Add bug reporter on 'module.json'
- Added midi-qol like a module dependency on 'module.json'
- Added 'fortunes Favor' , 'rage path of the totem warriori', 'favoured Enemy' , 'greater favoured Enemy', 'almost Dead', 'wounded', 'unconscious'. 
I will try in these day to make more internazionalization and management of the images.

### CODE REVIEW

File: scripts/effect-definitions.js [hunk 1]
```diff
@@ -84,759 +91,442 @@ export default class EffectDefinitions {
       this._encumbered,
       this._flanked,
       this._flanking,
+      this._favouredEnemy,
+      this._greaterFavouredEnemy,
       this._greatWeaponMaster,
       this._heavilyEncumbered,
       this._rage,
+      this._ragePathOfTheTotemWarrior,
```
Comment (DFreds) [inline scripts/effect-definitions.js:99]: This is already handled dynamically with the Rage effect. When `isDynamic` is set to true, it'll use the `dynamic-effects-adder.js` to see what additional effects it should add. I specifically handle Path of the Totem Warrior barbarians in the rage case in that class. Recommend removing this.

**Review by DFreds — CHANGES_REQUESTED** [review]
First off, I want to say thanks for the PR, wasn't really expecting anything like that. 

That being said, the reformatting I don't agree with. I use the default Prettier formatter in VSCode to format everything... and I intend to agree with what it chooses. Specifically, it breaks long lines to ensure developers with any screen size can split view files while still being able to see all the code. So in this PR, the formatting changes would need to be reverted.

If you use VSCode, just get the prettier extension and run the command "Format document with..." and choose Prettier.

I suppose I could add some default config file for prettier if you're not using VSCode.... but it's not a top priority for me at the moment.

File: scripts/effect-definitions.js (hunk 1 above)
Comment (p4535992) [reply scripts/effect-definitions.js]: Removed rage path of the totme warrior

**Comment by p4535992** [comment]
Do you want me to reformat the class 'effect-definitons'?

**Comment by p4535992** [comment]
Nevermind found it. Ty for the hint i didn't know that.

**Comment by DFreds** [comment]
Finished my first pass of the review. Let me know if you have questions

**Comment by DFreds** [comment]
Hm apparently I did set prettier to be single quotes instead of double quotes. You should, too, and maybe I'll look into that config file

**Comment by DFreds** [comment]
Added the prettier file to the main branch

**Comment by p4535992** [comment]
Close because of conflicts

**Pull request closed without merging**